apps/src/lib/util/audioApi.js

```diff
@@ -109,6 +109,60 @@ export const commands = {
     } else {
       Sounds.getSingleton().stopAllAudio();
     }
+  },
+  playSpeech(opts) {
+    apiValidateType(opts, 'playSpeech', 'text', opts.text, 'string');
+    apiValidateType(opts, 'playSpeech', 'gender', opts.gender, 'string');
+    apiValidateType(
+      opts,
+      'playSpeech',
+      'language',
+      opts.language,
+      'string',
+      OPTIONAL
+    );
+    const speechConfig = SpeechConfig.fromAuthorizationToken(
+      appOptions.azureSpeechServiceToken,
+      appOptions.azureSpeechServiceRegion
+    );
+    speechConfig.speechSynthesisOutputFormat =
+      SpeechSynthesisOutputFormat.Audio16Khz32KBitRateMonoMp3;
+
+    let voice = appOptions.azureSpeechServiceLanguages['English']['female'];
+    if (
+      appOptions.azureSpeechServiceLanguages[opts.language] &&
+      appOptions.azureSpeechServiceLanguages[opts.language][opts.gender]
+    ) {
+      voice =
+        appOptions.azureSpeechServiceLanguages[opts.language][opts.gender];
+    }
+    const synthesizer = new SpeechSynthesizer(speechConfig, undefined);
```

Comment:
For the literal here, add a comment with the parameter name that SpeechSynthesizer uses. i.e.
new SpeechSynthesizer(speechConfig, undefined /* inputName */);